docs/sharing-configurations.md

```diff
@@ -0,0 +1,183 @@
+---
+id: sharing-configurations
+title: Sharing configurations
+---
+
+In case you have many different projects, it can be helpful to have a **shared configuration** which can be used in all of them, instead of copy-pasting the same config for every project.
+
+This page explains how to create, publish and consume a shareable config.
+
+## Creating a Shareable Config
+
+Sharable configs are just [npm packages](https://docs.npmjs.com/about-packages-and-modules#about-packages) that export a single [prettier config file](./configuration.md).
+
+Before we start, make sure you have:
+
+- An account for [npmjs.com](https://www.npmjs.com/) to publish the package
+- Basic understating about [how to create a Node.js module](https://docs.npmjs.com/creating-node-js-modules)
+
+First, create a new package. We recommend creating a [scoped package](https://docs.npmjs.com/cli/v10/using-npm/scope) with the name `@username/prettier-config`.
+
+A minimal package should have at least two files. A `package.json` for the package configuration and an `index.js` which holds the shared prettier configuration object:
+
+```text
+prettier-config/
+├── index.js
+└── package.json
+```
+
+Example `package.json`:
+
+```json
+{
+  "name": "@username/prettier-config",
+  "version": "1.0.0",
+  "description": "My personal Prettier config",
+  "type": "module",
+  "exports": "./index.js",
+  "license": "MIT",
+  "peerDependencies": {
+    "prettier": ">=2.0.0"
```

Comment:
```suggestion
    "prettier": ">=3.0.0"
```

ESM config support was added in v3

Reply:
Done in https://github.com/prettier/prettier/pull/16495/commits/5ed397848304b886b7bf13f000c12de627331506 ✅